living/serbian.md

```diff
@@ -5,41 +5,73 @@ Additional names:
 - 
 
 News:
-- 
-- 
+- https://www.b92.net/ (Daily)
```

Comment:
Could this please be "Daily news" rather than just "Daily"? The descriptions should be sufficient without referring to the category name. Same for the other links below.